recipes/pyregion/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pyregion" %}
+{% set version = "1.2" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "4f57070564526974661bbbf96cd1d509a708909002dd34f0354ac80c391b5e61" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy 1.9.*  # [not (py36 and win)]
```

Comment:
Good point...this style of pinning is still new to me. @ocepaf, should I specify the numpy run requirements by platform? 

@synapticarbors -- the reason I went with the higher numpy version for building is that numpy 1.9 isn't available on Windows yet for python 3.6, and figured the minimum that would end up installed is 1.11 on python 3.6/Windows. 

But that is probably a really bad idea....